scripts/deploy.sh

```diff
@@ -78,6 +79,12 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
     # set network speed for vms
     echo "net_max_speed: 1000" >>$RPCD_VARS
 
+    # Copy ironic AIO config if ironic is enabled
+    if [[ "$DEPLOY_IRONIC" == "yes" ]]; then
+      cp -a -a ${OA_DIR}/etc/openstack_deploy/conf.d/ironic.yml.aio /etc/openstack_deploy/conf.d/ironic.yml
+      cp -a -a ${OA_DIR}/etc/openstack_deploy/env.d/ironic.yml /etc/openstack_deploy/env.d/
```

Comment:
Should we also be setting os_nova/defaults/main.yml:nova_virt_type to ironic as well?